wasm/src/lib.rs

```diff
@@ -353,6 +354,7 @@ pub fn run(
     event_cb: &js_sys::Function,
     shots: u32,
     language_features: Vec<String>,
+    profile: &str,
```

Comment:
This would likely be a breaking change for QCOM I guess if we change the API to run code.

Reply:
Actually, I don't think it is, as this is the wasm but the npm already handles overloading to two versions:
https://github.com/microsoft/qsharp/blob/swernli/histogram-respects-target/npm/qsharp/src/compiler/compiler.ts#L52-L64